apps/server/src/modules/deletion/uc/deletion-request.uc.spec.ts

```diff
@@ -0,0 +1,459 @@
+import { Test, TestingModule } from '@nestjs/testing';
+import { DeepMocked, createMock } from '@golevelup/ts-jest';
+import { setupEntities } from '@shared/testing';
+import { AccountService } from '@src/modules/account/services/account.service';
+import { ClassService } from '@src/modules/class';
+import { CourseGroupService } from '@src/modules/learnroom/service/coursegroup.service';
+import { CourseService } from '@src/modules/learnroom/service';
+import { FilesService } from '@src/modules/files/service';
+import { LessonService } from '@src/modules/lesson/service';
```

Comment:
Please replace all @src/modules/* to @modules/*
Please import services form @modules/account maybe you need create the service to 
~/modules/account/index.ts

Reply:
done